### PR TITLE
Dockerfile in root

### PR DESCRIPTION
**Pull request by azaitsau**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#{{{ demo_lti1p3
+FROM php:7.4-fpm-alpine as demo_lti1p3-base
+ENV MAKEFLAGS "-j8"
+
+RUN set -eux; \
+        cd /; \
+        apk add --no-cache --virtual .build-deps \
```
Comment (ekkinox) [inline Dockerfile:7]: if you check https://github.com/oat-sa/demo-lti1p3/blob/master/docker/phpfpm/Dockerfile#L9-L16, this is the only required extensions.

imo we should remove all the unused ones for faster and ligther builds, what doi you think ?

**Review by ekkinox — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (azaitsau) [reply Dockerfile:7]: Yes, you are right, I remove all unnecessary extensions, please check it

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            autoconf \
+            g++ \
+            unzip \
+            icu-dev \
+            sudo \
+            libmcrypt-dev \
+            libpng-dev \
+            librdkafka-dev \
```
Comment (ekkinox) [inline Dockerfile:15]: no need for librdkafka-dev

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            libxml2-dev \
+            linux-headers \
+            make \
+            oniguruma-dev \
+            postgresql-dev \
```
Comment (ekkinox) [inline Dockerfile:20]: no need for postgresql-dev

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        ;
+
+WORKDIR /var/www/html
+#}}} demo_lti1p3-base
+
+#{{{ composer
+FROM demo_lti1p3-base AS composer
+
+ARG SSH_KEY
+ENV SSH_KEY=$SSH_KEY
```
Comment (ekkinox) [inline Dockerfile:66]: all repos used in this app are public , no need to have SSH keys for composer

**Review by ekkinox — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (azaitsau) [reply Dockerfile]: done

File: Dockerfile (hunk 3 above)
Comment (azaitsau) [reply Dockerfile]: done

File: Dockerfile (hunk 4 above)
Comment (azaitsau) [reply Dockerfile]: removed

**Comment by ekkinox** [comment]
Cancelled due to @H4nks [PR](https://github.com/oat-sa/demo-lti1p3/pull/45)

**Pull request closed without merging**